Island_trope/Designer_guide/introduction_for_participatory_design_fiction.md

```diff
@@ -0,0 +1,114 @@
+[Designer guide](README.md)
+
+# Introduction for ___Participatory Design Fiction___ workshop
+
+```
+A getting started to your first workshop  
+```
+The ___Participatory Design Fiction___ workshop is going to put yourself into the role of a game master helping players to design an outcome to a fictive matter they want to address.  
```

Comment:
In this ___Participatory Design Fiction___ workshop you have the role of a game master - you are helping players to design an outcome to a fictitious matter they want to address.